.gitattributes

```diff
@@ -16,6 +16,7 @@
 *.sh eol=lf
 *.plist eol=lf
 MacOS_users_read_this_first.txt eol=lf
+triplea_maps.xml eol=lf
```

Comment:
Multiple reasons for LF in this config file:
- triplea_maps.xml is hosted on the game engine repo explicitly to *not* allow map admins/modders the ability to change the list of available maps used by the game engine. Thus only game engine developers will be using it.
- The CRLF in this file are unnecessary and gave me some trouble while maintaining the file.